src/ansi.rs

```diff
@@ -436,6 +442,23 @@ pub enum Attr {
     Background(Color),
 }
 
+/// Logical graphic character sets
+#[derive(Clone, Copy, Debug)]
+pub enum CharsetIndex {
+    /// Default set, is designated as ASCII at startup
+    G0,
+    G1,
+    G2,
+    G3,
+}
+
+/// Standard or common character sets which can be designated to G0-G3
+#[derive(Clone, Copy, Debug)]
+pub enum StandardCharset {
+    Ascii,
+    LineDrawing,
```

Comment:
It turns out to include more than just line drawing glyphs. The long name is usually "Special character and line drawing". I'm not a huge fan of long names in most cases, but it feels appropriate here (there are other line drawing + special character charsets).